index.md

```diff
@@ -52,9 +52,12 @@ layout: default
 ## License
 Lerna is released under the terms of the [Apache License Version 2.0](./LICENSE).
 
+\
```

Comment:
Could you add a comment why `\` exists in head of lines?

You can add comments using `<!-- ...  -->`.

Reply:
2203781d75fc888eaf5f2b81ece5f0d03be2fb52